admin/helpers/index.js

```diff
@@ -49,4 +49,5 @@ module.exports = function (app) {
   app.locals.deployVersion = appVersion
   app.locals.formatGdsDate = formatGdsDate
   app.locals.formatFullGdsDate = formatFullGdsDate
+  app.locals.guidancePdf = `${config.AssetPath}pdfs/mtc-administration-guidance-2018-03.pdf`
```

Comment:
Is this going to be a standard approach for all the resources such as the guidance pdf?

Reply:
@GeorgeChatzigiannis probably, yes.  it means we can hook in the asset domain easily at startup, when its used

Reply:
@GeorgeChatzigiannis - this solution improves on the feb-trial download solution as it means we can offload the download of this large file to a better server: e.g. nginx or a CDN rather than express.

As we are likely to change the guidance frequently I chose to resolve it via a variable, so that in the future it only needs changing in one place.

We can't just copy a new file over the old file, as this would prevent us from using far-future cache headers.